src/util/log.js

```diff
@@ -53,9 +57,17 @@ export const clearUserId = () => {
  * @param {Object} data Additional data to be sent to Sentry
  */
 export const error = (e, code, data) => {
-  if (Raven.isSetup()) {
-    const extra = { ...data, apiErrorData: responseApiErrorInfo(e) };
-    Raven.captureException(e, { tags: { code }, extra });
+  if (config.sentryDsn) {
+    const extra = { ...data, apiErrorData: responseApiErrorInfo(e), code };
+
+    Sentry.withScope(scope => {
+      Object.keys(extra).map(tag => {
+        return scope.setTag(tag, extra[tag]);
+      });
+      Sentry.captureException(e);
+    });
```

Comment:
I think this should use `setExtra` instead of just `setTag`:
```js
const extra = { ...data, apiErrorData: responseApiErrorInfo(e) };

Sentry.withScope(scope => {
  scope.setTag("code", code);
  Object.keys(extra).forEach(key => {
    scope.setExtra(key, extra[key]);
  });
  Sentry.captureException(e);
});
```
- Usage in [React example](https://docs.sentry.io/platforms/javascript/react/)
- [set_extra in unified API](https://docs.sentry.io/development/sdk-dev/unified-api/#scope)
- https://docs.sentry.io/enriching-error-data/scopes/?platform=browsernpm#local-scopes

Reply:
Change this part also in server/log.js!